admin_pages/general_settings/templates/state_details_settings.template.php

```diff
@@ -73,15 +73,15 @@
             <input class="STA_abbrev small-text "
                    id='STA_abbrev-XXX'
                    name="STA_abbrev_XXX"
-                   title=""
+                   aria-label=""
```

Comment:
I did some research and it is not recommended to have empty aria-label as its main use case is for screen readers.

At least, we can put the label title there.

Reply:
that makes sense, thnx
I'll have to just remove it then, because that is just a placeholder input for adding new content. It gets duplicated by JS and then populated with real data.

Reply:
fixed